presidio-image-redactor/tests/integration/test_dicom_image_pii_verify_engine_integration.py

```diff
@@ -0,0 +1,132 @@
+"""Integration test for dicom_image_pii_verify_engine
+
+Note we are not checking exact pixel data for the returned image
+because that is covered by testing of the "verify" function in
+the original parent ImagePiiVerifyEngine class.
+"""
+import PIL
+import pydicom
+import json
+import os
+
+from presidio_image_redactor.dicom_image_pii_verify_engine import (
+    DicomImagePiiVerifyEngine,
+)
+import pytest
+
+SCRIPT_DIR = os.path.dirname(__file__)
+PADDING_WIDTH = 25
+
+
+@pytest.fixture(scope="module")
+def mock_engine():
```

Comment:
That way, it's instantiated once for all test cases right? So one can affect the other? (although methods are immutable which is great)
I would just instantiate it once in each test on_test = DicomImagePiiVerifyEngine()

Reply:
As you mentioned, the methods are immutable so I believe having the one object should not be an issue.

I used this approach of instantiating once and passing `mock_engine` to each of the tests to follow what we did for the `DicomImageRedactorEngine` integration test.

Let's ask for another opinion. @omri374 , thoughts?

Reply:
This might be fragile as we can't ensure that the next test won't be mutable, and be a hell to debug :)
https://stackoverflow.com/questions/36579835/shared-context-between-net-unit-test-classes
You can also change the scope of the fixture for a quicker fix

Reply:
IMHO this is fine. While we cannot ensure that it would be immutable, this is definitely the design we have for the different engines in Presidio (all of them are stateless). 

If a test requires some change to the default, we can have them instantiated within the test instead of using the fixture.